paddle/fluid/operators/detail/grpc_server.cc

```diff
@@ -145,28 +145,36 @@ class RequestPrefetch final : public RequestBase {
         executor_(executor),
         program_(program),
         blkid_(blkid) {
+    request_.reset(new VariableResponse(scope, dev_ctx_));
     int method_id = static_cast<int>(detail::GrpcMethod::kPrefetchVariable);
-    service_->RequestAsyncUnary(method_id, &ctx_, &request_, &responder_, cq_,
-                                cq_, this);
+    service_->RequestAsyncUnary(method_id, &ctx_, request_.get(), &responder_,
+                                cq_, cq_, this);
   }
 
   virtual ~RequestPrefetch() {}
 
-  virtual std::string GetReqName() { return request_.varname(); }
+  virtual std::string GetReqName() { return request_->Varname(); }
 
   virtual void Process() {
     // prefetch process...
     ::grpc::ByteBuffer reply;
-    // TODO(Yancey1989): execute the Block which containers prefetch ops
 
-    VLOG(3) << "RequestPrefetch Process in";
+    std::string var_name = request_->OutVarname();
+    auto var_desc = program_->Block(0).FindVar(var_name);
+    framework::Scope* local_scope = &scope_->NewScope();
+    auto* var = local_scope->FindVar(var_name);
+    InitializeVariable(var, var_desc->GetType());
+
+    executor_->Run(*program_, local_scope, blkid_, false, false);
```

Comment:
If `executor_` is the member of `RequestPrefetch`, it will be created every time the request is send to the server, it's expensive, can make it the member of the server instance.

Reply:
Also can prepare it before run.

Reply:
The type of `executor_` is a pointer, so maybe we would create it for every request.
> Also can prepare it before run
A good idea and I will do that.

Reply:
Done.